runtime/caml/domain_state.tbl

```diff
@@ -129,11 +129,27 @@ DOMAIN_STATE(intnat, stat_major_collections)
 DOMAIN_STATE(intnat, stat_forced_major_collections)
 DOMAIN_STATE(uintnat, stat_blocks_marked)
 
-DOMAIN_STATE(uintnat, eventlog_startup_timestamp)
-DOMAIN_STATE(long, eventlog_startup_pid)
+/*****************************************************************************/
+/* Detailed stats */
```

Comment:
How have these reappeared? They were removed in https://github.com/ocaml-multicore/ocaml-multicore/pull/535